.pipelines/onebranch/pipeline.bootstrapper.pullrequest.yml

```diff
@@ -13,23 +13,26 @@ pr: none
 
 variables:
   Cdp_Definition_Build_Count: $[counter('', 0)] # needed for onebranch.pipeline.version task https://aka.ms/obpipelines/versioning
-  LinuxContainerImage: cdpxlinux.azurecr.io/global/ubuntu-1804-all:5.0   # Docker image which is used to build the project https://aka.ms/obpipelines/containers
+  LinuxContainerImage: cdpxlinux.azurecr.io/global/ubuntu-1804-all:5.0 # Docker image which is used to build the project https://aka.ms/obpipelines/containers
   Debian_Frontend: noninteractive
 
 parameters:
-- name: buildFluentBit
-  displayName: Build Fluent Bit?
-  type: boolean
+  - name: buildFluentBit
+    displayName: Build Fluent Bit?
+    type: boolean
```

Comment:
nit: While still valid yaml, this whitespace change isn't necessary and makes the review more tedious from what actually chaned.

Reply:
also nit: Is the `?` necessary in the display name for the `Build Fluent Bit?`